src/events/api/views.py

```diff
@@ -59,3 +59,10 @@ def get(self, request):
         response_data["schedule_day"] = settings.EVENTS_DAY_NAMES.items()
 
         return Response(response_data)
+
+
+class KeynoteEventListAPIView(views.APIView):
```

Comment:
Use ListAPIView instead of APIView
https://www.django-rest-framework.org/api-guide/generic-views/#listapiview